frontend/public/components/deployment-config.tsx

```diff
@@ -262,6 +264,20 @@ export const DeploymentConfigDetailsList = ({ dc }) => {
   );
 };
 
+export const DeploymentConfigDeprecationAlert: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Alert isInline variant="info" title={t('public~DeploymentConfigs deprecation')}>
+      <p>
+        {t(
+          'public~We will continue to support DeploymentConfigs until the 4.17 release, but we encourage you to migrate to Deployments as soon as possible.',
```

Comment:
What are your thoughts on something like:

```suggestion
          'public~Support for DeploymentConfigs will be removed from a future OpenShift Container Platform release, currently planned as OpenShift Container Platform 4.17. You should migrate to Deployments as soon as possible.',
```
In the docs we try and avoid a date or a specific future version if possible. We have gotten away with using "planned" before with supports approval.

Reply:
Yeah, here I would also rather avoid the dates, since we are not getting rid of the DC just deprecating and dropping support. That said I dont think we will be able to remove it from the UI even after 4.17.
That said I like the wording 👍 

cc'ing @ingvagabund for additional pair of eyes 👀

Reply:
DeploymentConfigs are going to be supported in every 4.X version. 4.14 is the first OCP version where we officially announce the deprecation (without removal) and recommend everyone to migrate to deployments wherever possible.

Reply:
Something along these lines:
```
DeploymentConfigs are getting deprecated in OpenShift Container Platform 4.14. We will continue to support DeploymentConfigs, but we encourage everyone to migrate to Deployments wherever it is possible.
```

Reply:
I like it. lets tweak it a bit with:
```
DeploymentConfigs are getting deprecated in OpenShift Container Platform 4.14. DeploymentConfigs will continue to be supported, but should be migrate to Deployments due to support.
```

Reply:
@opayne1 thoughts?

Reply:
What do you think of this? I was going off the suggestion I made in the [google doc](https://docs.google.com/document/d/1FXxHcsC8llE5IEFJmU3RTYDQBP9-dao718pEwAzKGQw/edit#heading=h.uyky4i3aaoaf) 

> Feature development of DeploymentConfigs will be deprecated in OpenShift Container Platform 4.14. DeploymentConfigs will continue to be supported for security and critical fixes, but you should migrate to Deployments wherever it is possible.

or 

> DeploymentConfigs will be deprecated in OpenShift Container Platform 4.14. DeploymentConfigs will continue to be supported, but you should migrate to Deployments due to support.

Reply:
@ingvagabund are we going to do security and critical fixes ?

Reply:
Then I would go with:
`Feature development of DeploymentConfigs will be deprecated in OpenShift Container Platform 4.14. DeploymentConfigs will continue to be supported for security and critical fixes, but you should migrate to Deployments wherever it is possible.`